ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -26,8 +28,13 @@ public IActionResult GetAll()
         }
 
         [HttpGet("{itemId}")]
+        [ItemValidationFilterAttribute]
```

Comment:
I think the filter should be in the controller level, as stated in the assignment and should support if the item ID is not available for the request, Thanks!

Reply:
updated